Install Failure on Win 11

## Description

Fresh install via go cli in PowerShell 7 on Windows 11 

## Steps to Reproduce

1. Open PowerShell
2. Run Command `go install github.com/zjrosen/perles@latest`
3. See error
```pwsh
go\pkg\mod\github.com\zjrosen\perles@v0.5.21\internal\orchestration\v2\handler\stop_process.go:134:15: undefined: syscall.Kill
```
## Expected Behavior

Successfully install the package.

## Actual Behavior

Error

## Screenshots

<img width="1154" height="161" alt="Image" src="https://github.com/user-attachments/assets/bd0db919-2285-4cf7-a55d-19f90e3667ed" />

## Environment

- **Perles version**: latest
- **Go version**: go1.25.5 windows/amd64
- **OS**: Win11 Pro 26200.7462
- **Terminal**:  PowerShell 7 hosted in Windows Terminal

## Additional Context

Add any other context about the problem here, including:
- I have it installed on another machine, but `perles -v` shows `perles version dev (commit: none, built: unknown)`

--- a/internal/orchestration/v2/handler/kill_process_unix.go
+++ b/internal/orchestration/v2/handler/kill_process_unix.go
@@ -1,0 +1,11 @@
+//go:build !windows
+
+package handler
+
+import "syscall"
+
+// killProcess forcefully terminates a process by PID using SIGKILL.
+// This is the Unix implementation.
+func killProcess(pid int) error {
+	return syscall.Kill(pid, syscall.SIGKILL)
+}

--- a/internal/orchestration/v2/handler/kill_process_windows.go
+++ b/internal/orchestration/v2/handler/kill_process_windows.go
@@ -1,0 +1,16 @@
+//go:build windows
+
+package handler
+
+import "os"
+
+// killProcess forcefully terminates a process by PID.
+// This is the Windows implementation using os.Process.Kill()
+// which calls TerminateProcess on Windows.
+func killProcess(pid int) error {
+	proc, err := os.FindProcess(pid)
+	if err != nil {
+		return err
+	}
+	return proc.Kill()
+}

--- a/internal/orchestration/v2/handler/stop_process.go
+++ b/internal/orchestration/v2/handler/stop_process.go
@@ -6,7 +6,6 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"syscall"
 	"time"
 
 	"github.com/zjrosen/perles/internal/orchestration/events"
@@ -16,11 +15,11 @@ import (
 )
 
 // GracefulStopTimeout is the maximum time to wait for graceful termination
-// before escalating to forceful termination (SIGKILL).
+// before escalating to forceful termination.
 const GracefulStopTimeout = 5 * time.Second
 
 // StopWorkerHandler handles CmdStopProcess commands.
-// It implements tiered termination: graceful (Cancel + timeout) then forceful (SIGKILL).
+// It implements tiered termination: graceful (Cancel + timeout) then forceful.
 // Follows the RetireProcessHandler pattern for lifecycle management.
 type StopWorkerHandler struct {
 	processRepo repository.ProcessRepository
@@ -126,12 +125,13 @@ func (h *StopWorkerHandler) gracefulStop(ctx context.Context, proc *repository.P
 	}
 }
 
-// forceStop immediately terminates the worker using SIGKILL.
+// forceStop immediately terminates the worker forcefully.
+// On Unix, this sends SIGKILL. On Windows, this calls TerminateProcess.
 func (h *StopWorkerHandler) forceStop(proc *repository.Process, liveProcess *process.Process) (*command.CommandResult, error) {
 	pid := liveProcess.PID()
 	if pid > 0 {
-		// Force kill the process using SIGKILL
-		_ = syscall.Kill(pid, syscall.SIGKILL)
+		// Force kill the process (platform-specific implementation)
+		_ = killProcess(pid)
 	}
 
 	return h.finishStop(proc, liveProcess, false)

